src/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -627,8 +628,11 @@ private void initTargetFolder() {
     public boolean onCreateOptionsMenu(Menu menu) {
         MenuInflater inflater = getMenuInflater();
         inflater.inflate(R.menu.main_menu, menu);
+        inflater.inflate(R.menu.sort_menu,menu.findItem(R.id.action_sort).getSubMenu());
```

Comment:
Please use spaces between parameters, there's many occurrences of this in the class.

Reply:
a checked in codeformatter in .idea directory can solve this out of the box